doc/docs.md

```diff
@@ -187,6 +211,16 @@ voidptr
 
 Please note that unlike C and Go, `int` is always a 32 bit integer.
 
+TBD: Type names in V currently permits snake_case/PascalCase/ALLCAP/SCREAMING_SNAKE_CASE styles, and distinguish them:
```

Comment:
@spytheman @lutherwenxu I wonder my understanding regarding the line is correct.

Reply:
`type` is used for making type aliases (like C's typedef), and there are no casing restrictions on the name after the `type` keyword. That is needed for C compatibility, so it is unlikely to change.

Reply:
So yes, I think what you wrote is true.